client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -28,10 +31,13 @@ export default function Dashboard(): JSX.Element {
   const loggedInUser: User | null | undefined = useAuth().loggedInUser;
   const [meetingOptions, setMeetingOptions] = useState<Meetings>([]);
 
-  const fetchMeetingsCallback = useCallback(async (id) => {
-    const meetings = await fetchMeetings(id);
-    if (meetings.success) setMeetingOptions(meetings.success.data);
-  }, []);
+  const fetchMeetingsCallback = useCallback(
+    async (id) => {
+      const meetings = await fetchMeetings(id, updateSnackBarMessage);
+      if (meetings.success) setMeetingOptions(meetings.success.data);
+    },
+    [updateSnackBarMessage],
+  );
```

Comment:
What made you use `useCallback` here?

Reply:
Hey Ethan! I had this in another PR, just updated it to make use of snackbars. I used useCallback because whenever we create an event, I want to fetch all the meetings from our DB again so it can populate the dashboard and show the event we just created. I thought it be a good choice because having the function memoized will prevent infinite re-renders since the alternative would be setting state in useEffect which would trigger this.